appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java

```diff
@@ -348,17 +367,17 @@ public void storeProjectSettings(String userId, long projectId, String projectSe
     String oldAccentColor = Strings.nullToEmpty(properties.getProperty("color.accent"));
 
     if (!newIcon.equals(oldIcon) || !newVCode.equals(oldVCode) || !newVName.equals(oldVName)
-      || !newUsesLocation.equals(oldUsesLocation) ||
-         !newAName.equals(oldAName) || !newSizing.equals(oldSizing) ||
-      !newShowListsAsJson.equals(oldShowListsAsJson) ||
+      || !newUsesLocation.equals(oldUsesLocation) || !newUsesCamera.equals(oldUsesCamera) ||
```

Comment:
I would suggest correcting the entirety of the if statement conditional to have `||` at the beginning of the line per Google style guide to be consistent because right now this is mixed (I think this is also on me).